js/app.js

```diff
@@ -0,0 +1,180 @@
+// 1
+function setAlarm(employed, vacation) {
+    return employed && !vacation;
+}
+
+// 2
+function closeCompare(a, b, margin = 0) {
+    if (b - margin <= a && a <= b + margin) {
+        return 0;
+    }
+    if (a < b) {
+        return -1;
+    }
+    return 1;
+
+// 3
+    function or(a, b) {
+        return !(!a && !b);
```

Comment:
Please get back to the lesson
<img width="748" alt="image" src="https://github.com/user-attachments/assets/9380eb96-c1da-4ca0-83b6-c0f5864a5f90">

Reply:
But the goal of this excercse was not to use || . Otherwise, it would have been much easier.

.... "When doing so, you cannot use the or operator: ||"....